plugins/lang-painless/src/main/java/org/elasticsearch/painless/PainlessLexer.java

```diff
@@ -1,19 +1,16 @@
 // ANTLR GENERATED CODE: DO NOT EDIT
 package org.elasticsearch.painless;
 
-import org.antlr.v4.runtime.CharStream;
+    import java.util.Set;
```

Comment:
We are going to have to figure out a way to make checkstyle ignore these.... Eventually!